testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js

```diff
@@ -869,70 +925,67 @@ QUnit.module('Scenarios', moduleConfig, () => {
         }).dxPivotGrid('instance');
 
         const expectedCells = [[
             { excelCell: { value: '', alignment: alignCenterTopWrap }, pivotCell: { alignment: 'left', colspan: 1, rowspan: 1, text: '', width: 100 } },
-            { excelCell: { value: null, alignment: undefined }, pivotCell: { area: 'column', colspan: 1, rowspan: 1, width: 100 } }
+            { excelCell: { value: null, alignment: undefined }, pivotCell: { area: 'column', colspan: 1, isLast: true, rowspan: 1, text: '', width: 100 } }
         ], [
             { excelCell: { value: 'A', alignment: alignLeftTopWrap }, pivotCell: { area: 'row', colspan: 1, dataSourceIndex: 1, isLast: true, path: ['A'], rowspan: 1, text: 'A', type: 'D' } },
             { excelCell: { value: null, alignment: undefined }, pivotCell: { colspan: 1, rowspan: 1 } }
         ], [
             { excelCell: { value: 'B', alignment: alignLeftTopWrap }, pivotCell: { area: 'row', colspan: 1, dataSourceIndex: 2, isLast: true, path: ['B'], rowspan: 1, text: 'B', type: 'D' } },
             { excelCell: { value: null, alignment: undefined }, pivotCell: { colspan: 1, rowspan: 1 } }
+        ], [
+            { excelCell: { value: 'Grand Total', alignment: alignLeftTopWrap }, pivotCell: { area: 'row', colspan: 1, isLast: true, rowspan: 1, text: 'Grand Total', type: 'GT', width: 100 } },
+            { excelCell: { value: null, alignment: undefined }, pivotCell: { colspan: 1, rowspan: 1 } }
         ]];
 
         helper.extendExpectedCells(expectedCells, topLeft);
 
         exportPivotGrid(getOptions(this, pivotGrid, expectedCells)).then((cellRange) => {
-            helper.checkRowAndColumnCount({ row: 3, column: 1 }, { row: 3, column: 2 }, topLeft);
+            helper.checkRowAndColumnCount({ row: 4, column: 1 }, { row: 4, column: 2 }, topLeft);
             helper.checkColumnWidths([excelColumnWidthFromColumn100Pixels, excelColumnWidthFromColumn100Pixels], topLeft.column);
             helper.checkFont(expectedCells);
             helper.checkAlignment(expectedCells);
             helper.checkValues(expectedCells);
             helper.checkMergeCells(expectedCells, topLeft);
             helper.checkOutlineLevel([0, 0, 0], topLeft.row);
             helper.checkAutoFilter(false, { from: topLeft, to: topLeft }, { state: 'frozen', ySplit: topLeft.row, xSplit: topLeft.column });
-            helper.checkCellRange(cellRange, { row: 3, column: 2 }, topLeft);
+            helper.checkCellRange(cellRange, { row: 4, column: 2 }, topLeft);
             done();
         });
     });
 
-    QUnit.test('Export [string(A,B) x None x number] & showColumnGrandTotals', function(assert) {
+    QUnit.test('Export [string(A,B) x None x None] & showRowGrandTotals & showColumnGrandTotals', function(assert) {
         const done = assert.async();
         const ds = {
             fields: [
                 { area: 'row', dataField: 'row1' },
-                { area: 'data', summaryType: 'count', dataType: 'number' }
```

Comment:
Unclear change: "area: data" is a part of the tested config.
Why was this line removed?